components/header/Header.tsx

```diff
@@ -1,26 +1,86 @@
 import React from 'react';
-import Search from './components/Search';
-import HeaderNav from './components/HeaderNav';
+import Link from 'next/link';
+import classNames from 'classnames';
+
+// Compoments
 import DotcmsLogo from './components/DotcmsLogo';
+import HamburgerButton from './components/HamburgerButton';
+import HeaderNav from './components/HeaderNav';
 import HeaderSelect from './components/HeaderSelect';
+import Search from './components/Search';
 
 export const Header = (): JSX.Element => {
+    const buttonIconClasses = [
+        'border-secondary',
+        'border',
+        'flex',
+        'focus:outline-none',
+        'items-center',
+        'justify-center',
+        'md:hidden'
+    ];
     return (
-        <header className="pt-4 px-6 bg-header border-b">
-            <div className="flex justify-left md:justify-center items-center mb-5 border-gray-100">
-                <div className="w-2/12">
+        <header className="md:h-32">
+            <div className="bg-purple border-gray-100 content-center flex flex-col justify-between md:flex-row md:h-full md:items-start md:justify-center md:pt-6 md:px-6">
+                <div className="flex h-20 justify-between md:h-full md:p-0 md:w-2/12 pt-6 px-6 w-full">
```

Comment:
You have to many height and width everywhere for a better responsive approach you need to make sure it adjust the content.